app/components/viral/pill_component.rb

```diff
@@ -14,19 +14,36 @@ class PillComponent < Viral::Component
       indigo: 'bg-indigo-100 text-indigo-800 dark:bg-indigo-900 dark:text-indigo-300',
       purple: 'bg-purple-100 text-purple-800 dark:bg-purple-900 dark:text-purple-300',
       pink: 'bg-pink-100 text-pink-800 dark:bg-pink-900 dark:text-pink-300',
-      primary: 'bg-primary-100 text-primary-800 dark:bg-primary-800 dark:text-primary-400'
+      primary: 'bg-primary-100 text-primary-800 dark:bg-primary-800 dark:text-primary-400',
+      transparent: 'text-slate-500 dark:text-slate-400'
+    }.freeze
+
+    BORDER_COLORS = {
```

Comment:
**nitpick:** Can you add a test to ensure that the border classes are applied properly?

Reply:
Added in f8168f65